test/benchee/formatters/html_integration_test.exs

```diff
@@ -5,42 +5,70 @@ defmodule Benchee.Formatters.HTMLIntegrationTest do
   @test_directory "test_output"
   @file_path "#{@test_directory}/my.html"
   @comparison_path "#{@test_directory}/my_comparison.html"
+
+  @default_test_directory "benchmark_output"
+  @default_file_path "#{@default_test_directory}/my.html"
+  @default_comparison_path "#{@default_test_directory}/my_comparison.html"
+
   test "works just fine" do
-    basic_test time: 0.01,
-               warmup: 0.02,
-               formatters: [&Benchee.Formatters.HTML.output/1],
-               formatter_options: [html: [file: @file_path]]
+    banchee_options = [time: 0.01, 
```

Comment:
Small typo here (although `banschee` would be a cool name).

Reply:
Looks like this typo is in a few places in this file. You might want to do a search and replace `s/banchee/benchee/g`

Reply:
yep, stupid mistake - it is corrected in new commit